### ISSUE TITLE
[BUG] M25 Park and M24 Unpark doesn't work the way it did in Marlin 1.1.9

### ISSUE THREAD

**Issue by hub-jba** [davidramiro/Marlin-Ai3M-2.0.x#5]
### Bug Description

It would seem that filament runout (and M600 colour change and user initiated pause) is not going to be so simple to implement in anycubic_TFT.cpp for Marlin 2.0. The way the head is parked and unparked from filament runout and user initiated pause has changed in marlin 2.0 for the **ADVANCED_PAUSE_FEATURE**

We need to rework all pause and restart behaviour to work with the new **ADVANCED_PAUSE_FEATURE** capability of Marlin 2.0.

I noticed that Marlin 2.0 has an extensible UI concept. It might enable us to integrate our anycubic_TFT code more deeply into the Marlin main loop and events/

See this as the original report: https://github.com/MarlinFirmware/Marlin/issues/12437.

### My Configurations

Unchanged from your repository master

### Steps to Reproduce
1. Apply fixes to issue #4 first, so that the filament runout or pause doesn't hang the system
2. Trigger a filament runout event
3. Wait for head to park
4. Load filament in the senor and accept the continue dialog.

**Expected behavior:**
1. The head should unpark and print job should continue

**Actual behavior:** [
1. Nothing happens the head doesn't move
#### Additional Information

From observing the terminal during a filament runout and continue it seem the M24 call doesn't do anything. Issuing an M108 down the terminal will actually cause the print head to continue and the printer to continue.

**Comment by hub-jba** [comment]
This was due to the M25 calling M125 which was waiting on user confirmation. The implementation with extensible ui api has methods specifically for this.
This PR re-implements this code using extensible ui: https://github.com/MarlinFirmware/Configurations/pull/154 and should make this issue redundant.
